contents/blog/transparent-enterprise-pricing.md

```diff
@@ -0,0 +1,125 @@
+---
+date: 2022-06-01
+title: Why we adopted transparent enterprise pricing (and you should too)
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/hog-in-a-room-generic.jpg
+featuredImageType: full
+categories: ["Inside PostHog"]
+author: ["simon-fisher", "Open source"]
+---
+
+> "I wonder how much this SaaS product costs to..."
+>
+> "Contact sales!"
+>
+> 😤
+
+Contact. Sales.
+
+Two words that embody everything we hate about modern SaaS products. And, when you see them, you know it's just the start of the sales dance.
+
+Even if you're deemed worthy enough for a sales rep to call you, it's likely you still won't find out the price until you've answered a _lot_ of questions.
+
+And maybe seen a demo.
+
+And perhaps your own hands-on evaluation.  
+
+And introduced your bosses' boss. 
+
+And consulted the legal department.
+
+And "hey, would you like to join our webinar?"
+
+It's all a colossal waste of time and energy that's better spent focusing on what matters: building better products.
+
+That's why, at PostHog, we've adopted transparent pricing and self-service for every paid plan, even PostHog Enterprise.
+
+> Simon Fisher leads customer success at PostHog. He's made a career out of helping companies automate stuff, including seven years at IBM and five years at Chef Software. He lives in Sheffield in the North of England with his wife, three young boys, a black labrador, and a killer Lego collection. Seriously, it's sweet af.
```

Comment:
I like the informality but 'sweet af' sounds a bit bro-y and not in keeping with our usual tone. Just 'sweet'?